CHANGELOG.md

```diff
@@ -28,7 +28,9 @@ The main work (all changes without a GitHub username in brackets in the below li
     -   Feature: Adds all elements (clusters, attributes, events, commands, device types and datatypes) introduced in Matter 1.2 and Matter 1.3.
 -   Matter-Core functionality:
     -   Feature: Increase Data Model revision to 17 (introduced by Matter 1.2)
+    -   Enhancement: Update Session parameters in PASE/CASE to match Matter 1.3 specification
```

Comment:
dummy .. Please leave this comment open ... Iwant to merge this PR and not squash it if we need to cherry pick stuff into a 0.9 branch or such :-)

Reply:
Hmm yes it would be nice if our PRs wouldn't squash by default, kind of throws away useful information